elastic/kibana
# [ResponseOps][Alerting] Store rule name in alerting event log documents

Resolves https://github.com/elastic/kibana/issues/139914

## Summary

Removes secondary query to lookup rule name and uses the value already populated in the event log

### Checklist


### To verify

- Logged in as elastic, create some dummy rules of type Index Threshold and Inventory
- Open the Logs tab and ensure the rule names populate under the Rule column
